WireUI/Sources/WireMainNavigationUI/Preview/MainTabBarControllerPreview.swift

```diff
@@ -36,7 +33,7 @@ private final class PlaceholderViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        view.backgroundColor = .yellow // ColorTheme.Backgrounds.surfaceVariant
+        view.backgroundColor = .yellow
```

Comment:
is it supposed to be yellow?

Reply:
No, this is just preview code (see the file name).
Sometimes I use colors for better visibility in previews and/or snapshot tests.
Example:
<img width="468" alt="Screenshot 2024-10-16 at 12 48 47 PM" src="https://github.com/user-attachments/assets/268fa454-2260-46f0-ad87-f5dd027ba1b5">